pages/login.tsx

```diff
@@ -19,6 +19,11 @@ function Login({ ssoPassEmploiEstActif, isFromEmail }: LoginProps) {
   const [errorMsg, setErrorMsg] = useState('')
   const router = useRouter()
 
+  async function handleSubmit(event: any, provider?: string) {
```

Comment:
```suggestion
  async function handleSignInSubmit(event: SyntheticEvent, provider?: string) {
```
nitpick: préciser le nom de la méthode + cf. https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/forms_and_events/